backend/content_iterator/task.py

```diff
@@ -3,10 +3,7 @@
 from ..processing_assets_model import ProcessingAssetType, ProcessingAssetsModel
 from ..types import JsonObject
 
-# From https://docs.aws.amazon.com/batch/latest/userguide/array_jobs.html
-# TODO: Set MAX_ITERATION_SIZE to 10_000 once we figure out [how to set a numeric
-# size](https://stackoverflow.com/q/66202138/96588)
-MAX_ITERATION_SIZE = 5
+MAX_ITERATION_SIZE = 10000
```

Comment:
Looking at where this is used:

- `any_item_index` should be renamed to `any_next_item_index`
- `should_return_zero_as_first_item_if_no_content` should use `any_item_count` instead of `any_item_index`